### ISSUE TITLE
Cannot launch on Manjaro 

### ISSUE THREAD

**Issue by benjamindaines** [RoPieee/roon-on-wine#5]
`0036:err:ntdll:RtlpWaitForCriticalSection section 0x7bd569c0 "../../../wine/dlls/ntdll/loader.c: loader_section" wait timed out in thread 0036, blocked by 0033, retrying (60 sec)
`
Cannot get Roon to start anymore, perhaps a wine update broke it?

**Comment by spockfish** [comment]
Any idea which version you're running?

**Comment by benjamindaines** [comment]
Got it working, had to update the install script to use a newer version of dotnet

`env WINEPREFIX=$PREFIX winetricks -q dotnet452`

**Comment by spockfish** [comment]
That was already done in the repo to 472 :-)
